scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -129,6 +129,12 @@
   (log-weird-state compute-cluster pod-name cook-expected-state-dict k8s-actual-state-dict)
   (kill-pod api-client name cook-expected-state-dict pod))
 
+(defn kill-pod-hard
+  "Kills the pod with a grace period of 0"
```

Comment:
This needs an explanation why we do this and when we do it and why we wouldn't do this everytime,